SpokestackTests/NLUTensorflowSlotParserTest.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  NLUTensorflowSlotParserTest.swift
+//  SpokestackTests
+//
+//  Created by Noel Weichbrodt on 2/25/20.
+//  Copyright © 2020 Spokestack, Inc. All rights reserved.
+//
+
+import Foundation
+import Spokestack
+import XCTest
+
+class NLUTensorflowSlotParserTest: XCTestCase {
+    
+    func testParse() {
+        let config = SpeechConfiguration()
+        config.nluVocabularyPath = SharedTestMocks.createVocabularyPath()
+        let parser = try! NLUTensorflowSlotParser(configuration: config, inputMaxTokenLength: 100)
+        let metaData = createMetadata().data(using: .utf8)
+        let metadata = try! JSONDecoder().decode(NLUTensorflowMetadata.self, from: metaData!)
+        
+        // selset parse
+        let taggedInputLocation = zip(["b_location"],["kitchen"])
+        let parsedSelset = try! parser.parse(taggedInput: taggedInputLocation, intent: metadata.intents.filter({ $0.name == "request.lights.deactivate" }).first!)
+        XCTAssertEqual(parsedSelset["location"]!.value as! String, "room")
+        
+        // integer parse
+        let taggedInputRating = zip(["b_rating"],["ten"])
+        let parsedInteger = try! parser.parse(taggedInput: taggedInputRating, intent: metadata.intents.filter({ $0.name == "rate.app" }).first!)
+        XCTAssertEqual(parsedInteger["rating"]!.value as! Int, 10)
+        
+        // digits parse
+        let taggedInputPhone = zip(["b_phone_number", "i_phone_number", "i_phone_number", "i_phone_number", "i_phone_number"],["42", "##38", "##34", "##17", "##45"])
```

Comment:
I think a test for the NLU component might be in order:
1) We're not exercising the code that would create the `taggedInput.*` data at runtime
2) Along those same lines, there's nothing that decodes a whole sequence (something that includes `o` tags and maybe more than one slot).